Core/HLE/sceKernelHeap.cpp

```diff
@@ -110,18 +110,65 @@ static u32 SysMemForKernel_536AD5E1()
 	return 0;
 }
 
+static int sceKernelFreeHeapMemory(int heapId, u32 block) {
+	u32 error;
+	KernelHeap* heap = kernelObjects.Get<KernelHeap>(heapId, error);
+	if (heap) {
+		if (block == 0)
+			INFO_LOG(SCEKERNEL, "sceKernelFreeHeapMemory(%d): heapId, block", heapId, 0);
+			return 0;
+		if (!heap->alloc.FreeExact(block)) {
+			ERROR_LOG(SCEKERNEL, "sceKernelFreeHeapMemory(%d): heapId, block Invalid pointer ", heapId, block);
+			return SCE_KERNEL_ERROR_INVALID_POINTER;
+		}
+		return hleLogSuccessInfoI(SCEKERNEL, 0, "sceKernelFreeHeapMemory(%d): heapId, block", heapId, block);;
```

Comment:
For all of these logs, the function name AND arguments are automatically included.  You only need to put the message here, no need to put `sceKernelFreeHeapMemory(%d)`.  Also, oops - `;;` at the end.

-[Unknown]